go.mod

```diff
@@ -257,3 +257,6 @@ replace gopkg.in/yaml.v3 => github.com/colega/go-yaml-yaml v0.0.0-20220720105220
 
 // We are using our modified version of the upstream GO regexp (branch remotes/origin/speedup-golang-1.19.2)
 replace github.com/grafana/regexp => github.com/grafana/regexp v0.0.0-20221005093135-b4c2bcb0a4b6
+
+// Replace goautoneg with a fork until https://github.com/munnerz/goautoneg/pull/5 is merged
+replace github.com/munnerz/goautoneg => github.com/charleskorn/goautoneg v0.0.0-20230303030534-7248a2f4c9cc
```

Comment:
Please make sure to make the same change to our internal enterprise repository, otherwise this change won't be reflected in our production :) (Given extra effort ... is it worth doing this?)

Reply:
Yep, done.